pycape/cape.py

```diff
@@ -271,11 +271,11 @@ async def _connect(self, function_ref):
             # Close the connection explicitly before throwing exception
             await self._ctx.close()
             raise RuntimeError(
-                f"No checksum received from enclave, expected{checksum}."
+                f"No function checksum received from enclave, expected{checksum}."
             )
 
         user_data_dict = json.loads(user_data)
-        received_checksum = user_data_dict.get("func_hash")
+        received_checksum = user_data_dict.get("func_checksum")
```

Comment:
this one can probably remain `"func_checksum"`, since it's an internal implementation detail